### ISSUE TITLE
Add verification that values of COG have not changed in COG examples notebook

### ISSUE THREAD

**Issue by zacdezgeo** [cloudnativegeo/cloud-optimized-geospatial-formats-guide#99]
We need to add a section in our COG examples notebook that demonstrates existing methods for verifying that data values remain unchanged post-conversion to COG (Cloud Optimized GeoTIFF) format. This addition will directly address recent queries about data integrity during format conversion.

Ensuring data consistency through conversion processes is crucial for maintaining the integrity of our datasets. To enhance user trust and provide clear guidance, it's important to showcase the methods we employ to verify that the values in our datasets stay consistent before and after being converted to COG format. Including these methods in the COG examples notebook will serve as a valuable reference for users and stakeholders.

cc: @wildintellect

**Comment by wildintellect** [comment]
@omshinde recently put together a good example of this on a project. Going to ask for a contribution to this guide.

**Comment by Rdataflow** [comment]
@wildintellect  maybe this comment https://github.com/geostandards-ch/cog-best-practices/issues/12#issuecomment-2979765083 is useful to demo unchanged values ?

**Comment by omshinde** [comment]
Yes @wildintellect .. [This](https://github.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/106) merged PR has a section validating data integrity post-conversion to COG. We can close this issue.
